src/component/Login.js

```diff
@@ -0,0 +1,120 @@
+import '../css/Login.css'
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Link from '@material-ui/core/Link';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: '100vh',
+  },
+  image: {
+    backgroundImage: 'url(https://community.mcgill.ca/wp-content/uploads/reporter.mcgill.ca141A7710b-2-33df7a956ac48acc56548c6c61a0118f440e48b0.jpg)',
```

Comment:
changes look good, but can load these images locally instead of from the internet? (mainly to improve speed and so we can reuse)

you can put them in src/img